openwisp_monitoring/check/classes/wifi_clients.py

```diff
@@ -0,0 +1,52 @@
+from django.utils import timezone
+from swapper import load_model
+
+from ...db import timeseries_db
+from .. import settings as app_settings
+from .base import BaseCheck
+
+AlertSettings = load_model('monitoring', 'AlertSettings')
+
+
+class WifiClients(BaseCheck):
+    def check(self, store=True):
+        values = timeseries_db.read(
+            key='wifi_clients',
+            fields='COUNT(DISTINCT(clients))',
```

Comment:
As discussed, please move the influxdb query language details to the timeseries DB backend.